src/HashBase.ts

```diff
@@ -21,23 +21,18 @@ interface BucketDictionary {
   }
 }
 
-export function openHashBase(
-  disklet: Disklet,
-  databaseName: string
-): Promise<HashBase> {
-  // check that the db exists and is of type HashBase
+export function openHashBase(disklet: Disklet, databaseName: string): HashBase {
```

Comment:
It would be helpful to know if the Disklet and database name are not a valid hash base. In such a case the user can then create the database.

Reply:
Since your change, the user would then have to do a query first